include/soci/column-info.h

```diff
@@ -96,11 +97,16 @@ struct type_conversion<column_info>
         }
         else if (type_name.find("blob") != std::string::npos ||
             type_name.find("BLOB") != std::string::npos ||
-            type_name.find("bytea") != std::string::npos ||
-            type_name.find("BYTEA") != std::string::npos)
+            type_name.find("oid") != std::string::npos ||
```

Comment:
I'm not sure why did `bytea` have to be changed to `oid`, is it related to the XML/long string changes? If so, I'd be curious to understand how.

Independently of my curiosity, I also wonder if this can/will have any compatibility implications?